### ISSUE TITLE
general hover states not working 

### ISSUE THREAD

**Issue by c-nv-s** [saadeghi/daisyui#626]
I would have expected that the ``hover:daisy-classname`` should work as normal but it doesn't seem to work at all.

e.g. 
```
<div class="w-16 border-4 hover:bg-accent hover:border-accent-focus hover:text-accent-focus">TESTING</div>
```
there is absolutely no change in the hover state. Is this intentional?

**Comment by saadeghi** [comment]
It does work.  
Example: https://play.tailwindcss.com/LqQkp3XP7c  

Are you sure your Tailwind setup is correct?

**Comment by saadeghi** [comment]
I see this is a problem on CDN file. I will look at it today. For now, you can use the installable node.js package or use one of sample projects on StackBlitz here: https://daisyui.com/docs/install/

**Comment by saadeghi** [comment]
Fixed in version `2.10.0`: 
CDN file:
```
https://cdn.jsdelivr.net/npm/daisyui@2.10.0/dist/full.css
```
Example: https://jsfiddle.net/saadeghi/dso7t1k3/1/
